config/initializers/logger.rb

```diff
@@ -0,0 +1,13 @@
+# suppresses routing errors
+if Rails.env.production?
+  class ActionDispatch::DebugExceptions
+    alias_method :old_log_error, :log_error
+    def log_error(env, wrapper)
+      if wrapper.exception.is_a?  ActionController::RoutingError
```

Comment:
__[RuboCop]__ <sub>posted by Sider</sub>

Style/GuardClause: Use a guard clause instead of wrapping the code inside a conditional expression.